rubocop-github.gemspec

```
@@ -20,7 +20,7 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = ">= 3.0.0"
 
-  s.add_dependency "rubocop", ">= 1.37"
+  s.add_dependency "rubocop", ">= 1.72"
   s.add_dependency "rubocop-performance", ">= 1.15"
   s.add_dependency "rubocop-rails", ">= 2.17"
```

It is also recommended to update the dependency versions of RuboCop Performance and RuboCop Rails.

```suggestion
  s.add_dependency "rubocop", ">= 1.72"
  s.add_dependency "rubocop-performance", ">= 1.24"
  s.add_dependency "rubocop-rails", ">= 2.23"
```

https://github.com/rubocop/rubocop-performance/releases/tag/v1.24.0
https://github.com/rubocop/rubocop-rails/releases/tag/v2.30.0

---

Thanks @koic! 🙇